docs/contract_call_parameters.md

```diff
@@ -72,6 +72,61 @@ In a list, `pair` and `union` values are always represented as objects: a `pair`
 
 The `methodsObject` method always takes a single object to represent the pair to be passed, while `methods` requires the pair fields to be spread. If annotations are present, they are used to identify the pair fielda in the corresponding properties of the JS object.
 
+## Bypassing the Michelson Encoder
+Users can bypass the `michelson-encoder` and `ContractAbstraction` by directly passing JSON Michelson in a `transfer` call. This eliminates the need to fetch and create a JS/TS contract abstraction using `tezos.wallet.at` or `tezos.contract.at` and also removes the requirement to create a local contract instance for interaction. As a result, the conversion of entrypoint parameters to the JSON Michelson format using the michelson-encoder is no longer necessary as used in the ContractAbstraction entrypoints as listed prior for `methods` and `methodsObject`.
+
+The `transfer` method can be used with both the `wallet` and `contract` providers. However, it is necessary to specify and set the provider accordingly, whether it is a `wallet` or `signer` provider.
+
+:::info
+Please Note:
+By using JSON Michelson directly in `transfer` calls with `Taquito`, developers can bypass potential edge cases that are not currently supported by the `michelson-encoder`. This is particularly useful when dealing with heavily nested entrypoint parameters involving multiple nest optional values.
+
+The michelson-encoder has limitations when encoding complex data structures with deep nesting in entrypoint parameters. By directly passing JSON Michelson, developers can freely construct and manipulate intricate entrypoint parameters without relying on the michelson-encoder to handle complex nesting. This approach provides more flexibility in working with complex data structures.
+:::
+
+A brief example for `Pair int string` using a Wallet Provider would be:
+
+```ts
+let nested2SomeNone1 = await tezos.wallet.transfer({ 
+  to: 'KT1NhJvzVTfidU5oV1NM9W9uwA2RSxNZz5Ai', 
+  amount: 0, 
+  parameter: { 
+    entrypoint: 'default', 
+    value: {
+      prim: 'Pair',
+      args: [
+        {int: 6}, 
+        {string:"tez"}
+      ],
+    }
+  }
+}).send()
+```
+
+Another example of Michelson type `pair (pair int nat) (option nat)` using the Contract Provider.
+
+```ts
+const nested2SomeNone2 = await tezos.contract.transfer({ 
+  to: 'KT1NhJvzVTfidU5oV1NM9W9uwA2RSxNZz5Ai', 
+  amount: 0, 
+  parameter: { 
+    entrypoint: 'default', 
+    value: {
+      prim: 'Pair',
+      args: [{
+        prim: 'Pair',
+        args: [
+          {int: 6},
+          {int: 7}
+        ],
+        }, 
+        { prim: 'None' } // <- this || { "int": 10 }
+      ],
+    }
+  }
+})
+```
+
 ## Map and big_map
```

Comment:
## Map and big_map section belongs to the prior part of the doc might want to move it above ## Bypassing the Michelson Encoder